content/nginx/deployment-guides/microsoft-azure/virtual-machines-for-nginx.md

```diff
@@ -71,32 +73,38 @@ In addition, to install NGINX software by following the linked instructions, you
 
    <span id="create-vm_Networking"></span>
-5. If you are creating VMs to use in [Active-Active HA for NGINX Plus on Microsoft Azure Using the Azure Standard Load Balancer]({{< ref "high-availability-standard-load-balancer.md" >}}), the two NGINX Plus VMs in that deployment must have public IP addresses with SKU type **Standard** instead of the default **Basic**.
+5. For [Active-Active HA for NGINX Plus on Microsoft Azure Using the Azure Standard Load Balancer]({{< ref "high-availability-standard-load-balancer.md" >}}):
 
-   For simplicity, we recommend allocating **Standard** public IP addresses for all six VMs used in the deployment. At the time of initial publication of this guide, the hourly cost for six such VMs was only $0.008 more than for six VMs with Basic addresses; for current pricing, see the [Microsoft documentation](https://azure.microsoft.com/en-us/pricing/details/ip-addresses/).
+ Two major virtual machines run NGINX Plus behind an Azure Load Balancer. And both VMs require **Standard** SKU public IP addresses. By default, Azure assigns **Basic** SKU IPs. So, you must manually change this during VM setup. 
 
-   To allocate a **Standard** public IP address, open the **Networking** tab on the **Create a virtual machine** window. Click <span style="color:#2d89d6; white-space: nowrap;">Create new</span> below the **Public IP** field. In the {{<nb>}}**Create public IP address**{{</nb>}} column that opens at right, click the **Standard** radio button under **SKU**. You can change the value in the **Name** field; here we are accepting the default created by Azure, {{<nb>}}**ngx-plus-1-ip**{{</nb>}}. Click the ** OK ** button.
+To avoid errors, assign **Standard** public IP addresses to the VMs you’ll use in the deployment. 
 
-   <a href="/nginx/images/azure-create-vm-networking.png"><img src="/nginx/images/azure-create-vm-networking.png" alt="screenshot of 'Networking' tab on Azure 'Create a virtual machine' page" width="1024" height="718" class="aligncenter size-full wp-image-64994" style="border:2px solid #666666; padding:2px; margin:2px;" /></a>
+When this guide was first published, the hourly cost for the six VMs was only $0.008. And this costlier than VMs with basic IP addresses. For current pricing, see the [Microsoft documentation](https://azure.microsoft.com/en-us/pricing/details/ip-addresses/).
+Depending on your work load, you might need up to 6 VMs. Follow these steps:
+  - Open the **Networking** tab on the **Create a virtual machine** window.
+  - Click  _Create new_ below the **Public IP** field.
+  - In the **Create public IP address column** that opens, click the **Standard Radio button** under **SKU**. 
+  - In the **Name** field, accept the default created by Azure, ngx-plus-1-ip. 
+  - Click the _Ok_  button.
```

Comment:
```suggestion
  - Select _Ok_.
```
When referring to a button by the label it is showing on screen there is no need to add "button" after. "Select Apply, Select Cancel, etc"